### DISCUSSION TITLE
Import Material

### DISCUSSION THREAD

**Discussion by kfateme** [NanoComp/meep#2845] [Q&A]
Hi all,

I want to optimize a splitter exactly like the example on the website: https://github.com/NanoComp/meep/blob/master/python/examples/adjoint_optimization/04-Splitter.ipynb. However, I want to import material from the Meep library. When I use the command 'from meep.materials import Si', I get the error 'RuntimeError: meep: simulation fields are NaN or Inf' after running the code.

I would appreciate it if you could help me.

Best regards,
Fatemeh

**Comment by smartalecH** [comment]
Take a look at this FAQ which describes how to overcome this common issue: https://meep.readthedocs.io/en/latest/FAQ/#why-are-the-fields-blowing-up-in-my-simulation
